app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    goal = split_integer(16, 4)
+    assert sum(goal) == 16
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    goal = split_integer(6, 2)
+    assert goal[0] == goal[1]
```

Comment:
Add an error message(for all tests). It's good when the user understands what the problem is